README.md

```diff
@@ -52,4 +50,4 @@ Also, you can of course not use some features outlined here. That's why the whol
 * [`borgcron_start.sh`](borgcron_start.sh) – Header file with config options for a specific backup. This calls [`borgcron.sh`](borgcron.sh) internally.
 * [`checklastbackup.sh`](checklastbackup.sh) – Script, which you can execute at login (Add to your `.bashrc` file or so), which notifies you when a backup has failed. Otherwise it remains silent.
 * [`cronsizecache.sh`](cronsizecache.sh) – Small one-liner to cache the size of the dir where backups are stored. (useful for remote backup servers) You can then include the result with `cat` in your login script.
-* [`databasedump.sh`](databasedump.sh) – Dumps your database into a dir/file. Make sure, this script and the dump dir is only readable by your backup process. Can be executed with higher privileges (i.e. root) brefore creating the backup than your backup script afterwards.
+* [`databasedump.sh`](databasedump.sh) – Dumps databases into a dir/file. Make sure, that this script and the dump dir are only readable by your backup user. Script might have to be executed with higher privileges (i.e. root) for creating the backup.
```

Comment:
Just FYI, you do not need to reply to every comment, I'll see when you've changed the things. (And a better tick icon would be the Emoji :white_check_mark: – `:white_check_mark:`, but does not matter…)